lib/ruby-push-notifications/gcm/gcm_connection.rb

```diff
@@ -27,8 +27,9 @@ class GCMConnection
       # @param notification [String]. The text to POST
       # @param key [String]. The GCM sender id to use
       #    (https://developer.android.com/google/gcm/gcm.html#senderid)
+      # @param optional options [Hash]. Options for the HTTP connection.
```

Comment:
Same as before. It would be helpful to add a list of the valid values and defaults